src/modules/commander/Commander.cpp

```diff
@@ -839,7 +839,9 @@ Commander::handle_command(vehicle_status_s *status_local, const vehicle_command_
 
 						// Refuse to arm if in manual with non-zero throttle
 						if (cmd_arms
-						    && (status_local->nav_state == vehicle_status_s::NAVIGATION_STATE_MANUAL
```

Comment:
Isn't that on purpuse for spring-loaded throttle sticks?

Reply:
Yep. I don't think we can do this change easily because flying down and yawing is "normal". We would need to be able to trust the land detector 100% that it never gets it wrong in that condition AND that the land detector is enabled. On a quick pass I didn't see this enforced and so this patch would lead to in-air disarms.

Reply:
@LorenzMeier I think you mean something else: arming using sticks.

This does not trigger arming, this is just a protection that doesn't let you arm via MAVLink if the throttle stick is high.

@bkueng what is the value of a spring loaded throttle stick at center? If it is 0.5, this won't let you arm unless you push the stick down. If the throttle stick went from -1 to 1 it would work :thinking:. I guess for POSCTL and ALTCTL we could check for `> 0.6` instead of `> 0.1` as it is now.

Reply:
> @bkueng what is the value of a spring loaded throttle stick at center? If it is 0.5, this won't let you arm unless you push the stick down. If the throttle stick went from -1 to 1 it would work thinking. I guess for POSCTL and ALTCTL we could check for > 0.6 instead of > 0.1 as it is now.

Yes 0.5.

Reply:
Ok, I'll add a check for 0.6 then.

Reply:
@LorenzMeier please check my comment above and/or review again.

Reply:
@LorenzMeier bump